_projects/host-generic-pc/build.project

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Shell script for building host-generic-pc project
+#
+# Copyright 2022 Phoenix Systems
+# Author: Lukasz Kosinski
+#
+
+[ "${BASH_SOURCE[0]}" -ef "$0" ] && echo "You should source this script, not execute it!" && exit 1
+
+. "_targets/build.project.host-generic"
+
+b_image_project () {
+	b_log "The images have been built for the host-generic-pc platform"
```

Comment:
I think information about platform name can be fetched from variables, this will save some editing when making these files

Reply:
Done, used TARGET variable.